op-node/rollup/derive/l1_retrieval_test.go

```diff
@@ -118,7 +119,7 @@ func TestL1RetrievalNextData(t *testing.T) {
 		{
 			name:         "simple retrieval",
 			prevBlock:    a,
-			sysCfg:       eth.SystemConfig{BatcherAddr: common.Address{0x55}},
+			sysCfg:       eth.SystemConfig{BatcherAddr: common.Address{0x55}, BatcherHashVersion: 0},
```

Comment:
So, currently this tests that we haven't broken the V1 behavior. This is an easy point to introduce a V2 test as well, in the next step.